lib/facility_factory.rb

```diff
@@ -0,0 +1,36 @@
+class FacilityFactory
+  
+  def initialize
+  end
+
+  def normalize_phone(phone_string)
+    if phone_string != nil && !phone_string.start_with?('(')
+      phone_string.insert(0, '(').insert(4, ')').insert(5, ' ').insert(9, '-')
+    else return phone_string
+    end
+  end
```

Comment:
Nice job here! I like that you've got another method to help format your phone number.